loafwallet/src/Strings/tr.lproj/Localizable.strings

```diff
@@ -232,1169 +232,1169 @@
 /* Regular fee */
 "FeeSelector.regular" = "Düzenli";
 
-/* Fee Selector regular fee description */
-"FeeSelector.regularLabel" = "Estimated Delivery: 2.5 - 5+ minutes";
+/* Ücret Seçici normal ücret açıklaması */
+"FeeSelector.regularLabel" = "Tahmini Teslimat: 2,5 - 5+ dakika";
 
-/* Fee Selector title */
-"FeeSelector.title" = "Processing Speed";
+/* Ücret Seçici başlığı */
```

Comment:
@JasperHeist Oh I see its confusing but the process is to leave the Comments in english (revert line 238) for example. Line 239 is correct.  Meaning only the content on the right side of the label name.